clusterloader2/pkg/config/templateprovider.go

```diff
@@ -29,13 +30,15 @@ import (
 // TemplateProvider provides object templates. Templates in unstructured form
 // are served by reading file from given path or by using cache if available.
 type TemplateProvider struct {
+	basepath      string
```

Comment:
Please add newline above the lock's block.

Reply:
done.